staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -305,6 +312,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	StructuredAuthenticationConfiguration: {Default: false, PreRelease: featuregate.Alpha},
 
+	StructuredAuthorizationConfiguration: {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
also register this into pkg/features/kube_features.go with the other feature gates we inherit from genericfeatures